engines/decidim_hospitalet-surveys/app/commands/decidim_hospitalet/surveys/admin/create_survey_result.rb

```diff
@@ -89,6 +89,13 @@ def existing_user
           ).first
         end
 
+        def default_user
+          Decidim::User.where(
```

Comment:
The worst thing it will happen is that the proposal won't have an assigned user, which it shouldn't happen if you run the migrations.